test/test-pages/rtl-4/expected-metadata.json

```diff
@@ -2,6 +2,6 @@
   "title": "RTL Test",
   "byline": null,
   "excerpt": "Lorem ipsum dolor sit amet.",
-  "dir": "rtl",
+  "dir": null,
```

Comment:
We also use the detected direction (ie in the metadata we extract) for the markup for the title, subtitle etc., right? So it makes a difference there. I think we should probably post-process the final readability result and de-nest any singly-nested container nodes. That would also tidy up the markup, and we could preserve any directionality when doing this. Let's do this in a followup as well, though. I'll file a separate issue for this.